### ISSUE TITLE
important code error: losses["loss"].detach()  in train_util.py line 246

### ISSUE THREAD

**Issue by Frank21st** [WuJunde/MedSegDiff#39]
please check the losses variable

            if isinstance(self.schedule_sampler, LossAwareSampler):
                self.schedule_sampler.update_with_local_losses(
                    t, losses["loss"].detach()
                )
            losses = losses1[0]
            sample = losses1[1]
![B](https://user-images.githubusercontent.com/45085708/221464145-500e8d30-15b5-4f61-b932-32260b0a7f13.png)

**Comment by heha102** [comment]
I met the same situation. Do you know how to solve it now?

**Comment by Frank21st** [comment]
waiting for the author to reply...

**Comment by WuJunde** [comment]
you can simply put line 248 in front of it. actually this block will not be run in the current version

**Comment by Frank21st** [comment]
losses["loss"].detach()  ---> losses1[0].detach() ?
